src/sqlfluff/dialects/dialect_sparksql.py

```diff
@@ -386,6 +393,21 @@
         # there since there are no significant syntax changes
         "JDBC",
     ),
+    AsOfGrammar=Sequence("AS", "OF"),
```

Comment:
Not sure this is really necessary. Only use it twice and the grammar shortcut is longer than than the actual grammar! That can make sense if want to override it in other dialects but not so sure here.